src/types.js

```diff
@@ -36,6 +36,26 @@ export function isArray(value) {
 }
 
 /**
+ * Converts an array-like object to an array.
+ * @param {?IArrayLike<*>|string} arrayLike
+ * @return {!Array.<*>}
+ */
+export function toArray(arrayLike) {
+  if (!arrayLike) {
+    return [];
+  }
+  const length = arrayLike.length;
+  if (length > 0) {
+    const array = new Array(length);
+    for (let i = 0; i < length; i++) {
```

Comment:
Cached `length` iteration is actually slower than `i < arrayLike.length` in current browsers.